### DISCUSSION TITLE
LINK GENERATOR

### DISCUSSION THREAD

**Discussion by djmakers** [rejetto/hfs#537] [Ideas]
**Will be fine to replace this command "Get list" with command "Get Link"  
Thank you!**
![image](https://github.com/rejetto/hfs/assets/47154419/34901039-f219-4796-80ae-d538e84313ac)

**I see more fine linke this example below.**
![image](https://github.com/rejetto/hfs/assets/47154419/e9baa957-a757-48a9-80d9-0559bb455952)

**And ofcourse I need a option to enable or disable "Time Stamp"**
![image](https://github.com/rejetto/hfs/assets/47154419/745d968e-0a48-4c7f-ab99-181a700a54f8)

**Comment by BussyBakks** [comment]
bruh clone this repo to ur computer, edit as more as u know and build it 🗣

**Reply by djmakers** [reply]
I would edit, but I don't understand how to compile the application at the end with exe executor, if you can help me with a video tutorial, or any help would be welcome, thanks.

When finishing the project, I don't understand how to compile the files to look like the picture below.


![image](https://github.com/rejetto/hfs/assets/47154419/41d8fbf3-5453-472d-8894-c9f4fa606b41)

**Reply by rejetto** [reply]
you can do already customize a lot of with css and javascript.
i don't suggest cloning, or you'll have to work to keep up with the updates, and if you run a server it's important to stay up to date for security.

**Comment by rejetto** [comment]
> Will be fine to replace this command "Get list" with command "Get Link"

you want to just change the name or the function?

if you want to remove  a command you can use css (use inspect element) or javascript (refer to [documentation for event fileMenu](https://github.com/rejetto/hfs/blob/main/dev-plugins.md)) whatever you like best.

> And ofcourse I need a option to enable or disable "Time Stamp"

you can already do it with css, but it's a bit complicated a bit unreliable. I want to introduce dedicated id in next version, so it's easier.

> I want to edit more elements, but I don't have access to entire code :(

say what you want to change and we'll see

**Reply by djmakers** [reply]
It would be good to add   **Copy link folder** and **copy link file** in the context menu, or maybe replace it with **get** list, which I don't see in **get list** as a necessary function!

**Comment by rejetto** [comment]
i guess this is the same request that you opened an issue #541 , so i replied there.
i understand that you don't need "get list", but that says little on how many people uses it. I have no idea to be honest, but that was a feature available in hfs 2 so i considered good including it.
